domains/fishyfish.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "ArduFish123",
+           "email": "vlad.robert.juganaru@gmail.com",
+           "discord": "619521237777121280"
+        },
+    
+        "record": {
+            "TXT": "amos.ns.cloudflare.com"
```

Comment:
Wrong records for NS records, also we don't support them at the moment. Even if we did it would only be for maintainers and trusted users most likely.